includes/views/html-view-layout-popup.php

```diff
@@ -9,6 +9,11 @@
 
 $post_id = is_front_page() && ! is_page() ? '-1' : get_the_ID();
 
+global $post;
+
+if( get_post_meta($post->ID, 'enable_disable_ess', true) !== 'yes'){
```

Comment:
Follow WordPress Coding Standard guideline for Yoda condition https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions